HomeTask1/Program.cs

```diff
@@ -1,3 +1,133 @@
-﻿// See https://aka.ms/new-console-template for more information
-string name = Console.ReadLine();
-Console.WriteLine("Hello," + name + "!");
+﻿using System;
+using System.Transactions;
+using System.Xml.Linq;
+
+namespace MyApp
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Lesson1_Task1();
+            Lesson2_Task1_Calculator();
+            Lesson2_Task2_RangeDetermination();
+            Lesson2_Task3_RangeDetermination();
+            Lesson2_Task4_ParityCheck();
+            Lesson2_Task4_1_ParityCheck();
+        }
+        public static void Lesson1_Task1()
+        {
+            string? name = Console.ReadLine();
+            Console.WriteLine("Hello," + name + "!");
+        }
+        public static void Lesson2_Task1_Calculator()
+        {
+            double operand1 = 10.123;
+            double operand2 = 6.789;
+            double result = 0;
+            Console.Write("Enter sign (+,-,*,/): ");
+            string? sign = Console.ReadLine();
+            switch (sign)
+            {
+                case "+":
+                    result = operand1 + operand2;
+                    break;
+                case "-":
+                    result = operand1 - operand2;
+                    break;
+                case "*":
+                    result = operand1 * operand2;
+                    break;
+                case "/":
+                    if (operand2 == 0)
+                    {
+                        Console.WriteLine($"Division by 0 is not allowed");
+                        return;
+                    }
+                    else
+                    {
+                        result = operand1 / operand2;
+                    }
+                    break;
+            }
+            Console.WriteLine($"{operand1} {sign} {operand2} = {result}");
+        }
```

Comment:
empty line

Reply:
Excuse me, where is it?

Reply:
before and after method should be empty line